<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>硬件参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>载波信号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>频率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zaibo_freq"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>GHz</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>功率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zaibo_power"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>dBm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>调制信号</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>幅值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tiaozhi_amp"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>   V   </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>偏置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tiaozhi_bias"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>   V   </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>调制波形</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="zhengbanbo">
         <property name="text">
          <string>正半波</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="juxingbo">
         <property name="text">
          <string>矩形波</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="zhengxianbo">
         <property name="text">
          <string>正弦波</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>281</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>环境参数（对应实验选填）</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>251</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>辐照距离</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="distance"/>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>m</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>转棒实验</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>转速    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rotate_v">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>r/min</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>加速度</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rotate_a">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>r/min</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>游泳实验</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>水温   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="water_temp">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>℃</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>运动轨迹检测场景大小</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>长   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hight">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>cm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>宽   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="widith">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>cm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="tijiao">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
